general/utils.go

```diff
@@ -0,0 +1,27 @@
+package general
+
+import (
+	"github.com/jfrog/jfrog-cli-core/v2/common/commands"
+	"github.com/jfrog/jfrog-cli-core/v2/utils/config"
+	"github.com/jfrog/jfrog-client-go/utils/errorutils"
+	"net/url"
+	"strings"
+)
+
+// Deduce the server ID from the URL and add server details to config.
+func ConfigServerWithDeducedId(server *config.ServerDetails, interactive, webLogin bool) error {
+	u, err := url.Parse(server.Url)
+	if errorutils.CheckError(err) != nil {
+		return err
+	}
+	// Take the server name from host name: https://myjfrog.jfrog.com/ -> myjfrog
+	serverId := strings.Split(u.Host, ".")[0]
```

Comment:
I'm not sure it will work when providing an IP.
Can we use a default name in this case?

![image](https://github.com/jfrog/jfrog-cli-core/assets/11367982/51270cf5-0c27-46b2-9222-eaf50e0be7aa)